test/tools/ossfuzz/solProtoFuzzer.cpp

```diff
@@ -202,7 +202,7 @@ DEFINE_PROTO_FUZZER(Program const& _input)
 		// With libFuzzer binary run this to generate a YUL source file x.yul:
 		// PROTO_FUZZER_DUMP_PATH=x.yul ./a.out proto-input
 		ofstream of(dump_path);
-		of.write(sol_source.data(), sol_source.size());
+		of.write(sol_source.data(), static_cast<unsigned>(sol_source.size()));
```

Comment:
I see in the docs that `write()` expects [std::streamsize](https://en.cppreference.com/w/cpp/io/streamsize) which is signed. So why does `unsigned` even work here? Is that because it's usually 32-bits while `streamsize` is usually 64-bit long?

Anyway, I'd use `streamsize` for correctness here though in practice unsigned is probably enough for any data we may write:

```suggestion
		of.write(sol_source.data(), static_cast<streamsize>(sol_source.size()));
```